netcore/NetCore.Session.IntegrationTests/NetCore.Session.IntegrationTests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+
+    <Platforms>x64</Platforms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.5.0" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.3">
+      <PrivateAssets>all</PrivateAssets>
```

Comment:
Strikes me that all the test assets are all the same, so why is the runner special?

Reply:
Not sure what you're asking here. Are you wondering why xunit.runner.visualstudio is at 2.4.3 and xunit is at 2.4.1? If so, 2.4.1 is the latest version of xunit on  NuGet.org.

Reply:
You have this one specifically tagged with this `PrivateAssets` tag but none of the others.